assets/wire/components/fields/DurationLabel.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {gettext} from 'utils';
+import {get} from 'lodash';
+
+export function DurationLabel ({item, filterGroupLabels}: any) {
+    if (item.extra?.duration == null) {
+        return null;
+    }
+
+    function zeroPad(number: number) {
+        if (number.toString().length < 2) {
+            return `0${number}`;
+        } else {
+            return number;
+        }
+    }
+
+    function secondsToHHMMSS(seconds: number) {
+        seconds = Number(seconds);
+
+        const h = zeroPad(Math.floor(seconds / 3600));
+        const m = zeroPad(Math.floor(seconds % 3600 / 60));
+        const s = zeroPad(Math.floor(seconds % 3600 % 60));
+
+        const value = `${h}:${m}:${s}`;
+    
+        return value;
+    }
+
+    const label = get(filterGroupLabels, 'duration', gettext('Duration'));
+    const time = secondsToHHMMSS(item.extra.duration);
+    const text = label + ': ' + time;
+
+    return (
+        <span>{text}</span>
```

Comment:
we have some classes on the other fields, shouldn't we use those here as well?

Reply:
we just use classname for UrgencyLabel in some specific case